content/Solutions/Optimizations/STIGCompliance.md

```diff
@@ -0,0 +1,273 @@
+Encrypting at the dataset level allow more granular control over encrypted verses unencrypted datasets, and allows setting the encryption type to either key or passphrase protection.
+Child datasets of encrypted datasets inherit encryption from the parent dataset.
+See [Storage Encryption]({{< relref "EncryptionSCALE.md" >}}) for more information on encrypting datasets.
+
+Datasets with encryption allow users to lock the datasets to prevent reading from or writing to the dataset until it is unlocked.
+
+### Data Transfer Security
+TrueNAS allows encryption on data transfers made through cloud sync and replication tasks.
+This adds a layer of encryption on top of dataset-level encryption whether the dataset is encrypted or not.
+
+See [Encrypting Cloud Sync Tasks]({{< relref "/SCALE/SCALETutorials/dataprotection/CloudSyncTasks/_index.md" >}}) for more information on encrypting cloud sync tasks, or [Adding Transfer Encryption]({{< relref "/scale/scaletutorials/dataprotection/replication/advancedreplication/_index.md" >}}) for information on adding encryption to remote replication tasks.
+
+#### Maintain SSH Connection Credentials
+TrueNAS uses SSH connection credentials for cloud backup tasks, cloud sync tasks, rsync tasks, and replication tasks.
+Purge the list of SSH connection to backup servers no longer actively useds to minimize security vulnerabilities through connections to the remote servers or services no longer used.
+Select out-of-date or inactive SSH connections, SSH Keypair, or cloud credential to edit or delete it.
+
+### Updating TrueNAS Releases
+TrueNAS provides a way to update to the latest releases from the UI. The main Dashboard and Software > Updates screen provide access to the latest releases for the currently installed release train selected in TrueNAS.
+Alternatively, users can use an <file>iso</file> to clean install the latest version of TrueNAS.
+Each release upgrade creates a new boot environment.
+
+#### Manage Boot Environments
+To manage versions of TrueNAS releases, go to **System > Boot** to open the **Boot Environments** screen.
+Select the checkbox(es) for releases you want to delete from the list of inactive releases.
+Maintaining releases does not pose security risks but does consume space on the boot pool.
+See [Boot Pool Management]({{< relref "ManageBootEnvironSCALE.md" >}}) for more information on working with boot pool environments.
+
+### Virtualization
+This document does not cover the virtual machine envirnoments created by users.
+STIG compliance for these users-deployed environments is based on the operating system and applications deployed in these VMs.
+
+## Future STIG Compliance
+TrueNAS does not comply with STIG for the following findings that are planned for a future release:
```

Comment:
Should probably call out the specific STIG that we don't comply with here (GPOS STIG). 
(Not complying with STIGs in general isn't really a thing.)

Reply:
added particulars here in next commit.